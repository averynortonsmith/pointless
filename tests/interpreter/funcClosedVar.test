
output = [a, b, c, d] |> println

foo(n) = (x) => y where y = n
a = foo(123)
b = foo(false)
c = [a, b].!getHead("123")
d = [b, a].!getHead("123")