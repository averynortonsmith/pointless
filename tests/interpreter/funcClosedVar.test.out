[PtlsFunc, PtlsFunc, 123, false]
